packages/lib/src/core/Context/CoreProvider.tsx

```diff
@@ -34,7 +37,12 @@ class CoreProvider extends Component<CoreProviderProps> {
         if (this.state.loaded) {
             return (
                 <CoreContext.Provider
-                    value={{ i18n: this.props.i18n, loadingContext: this.props.loadingContext, commonProps: this.props.commonProps || {} }}
+                    value={{
+                        i18n: this.props.i18n,
+                        loadingContext: this.props.loadingContext,
+                        commonProps: this.props.commonProps || {},
+                        resources: this.props.resources
+                    }}
```

Comment:
Did you also consider not attaching the `resources` module to the `CoreProvider` module but rather create its own `ResourcesProvider` mechanism? Same approach as the `SRProvider`, in separating the concerns.

This is kinda a a big change for the PR, I know.. but just shooting out the idea as we are already touching basically all Components using the Provider.

We could create a high level `AdyenCheckoutProvider` Component which contains `CoreContextProvider`, `SRPanelProvider`, and any other providers that we would need now/in the future. 

So, in our Components, we would just render the elements within `<AdyenCheckoutProvider>`